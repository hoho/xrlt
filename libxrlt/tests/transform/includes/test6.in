id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:1, type:600, last:0, error: 0, data:hello
id:1, type:600, last:0, error: 0, data:
id:1, type:600, last:0, error: 0, data:world
id:1, type:600, last:0, error: 0, data:
id:1, type:600, last:1, error: 0, data:
id:2, type:600, last:0, error: 0, data:hello
id:2, type:600, last:0, error: 0, data:
id:2, type:600, last:0, error: 0, data:world
id:2, type:600, last:0, error: 0, data:
id:2, type:600, last:1, error: 0, data:
id:0, type:0, last:0, error: 0, data:
