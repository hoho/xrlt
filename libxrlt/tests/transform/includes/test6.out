XRLT_STATUS_CHUNK
chunk: 11abcsdafkjhl
XRLT_STATUS_SUBREQUEST
sr id: 1
sr method: GET
sr type: TEXT
sr url: /sub2
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_SUBREQUEST
sr id: 2
sr method: GET
sr type: TEXT
sr url: /sub
sr query: test=hello&test=123
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_CHUNK
chunk: ahelloworldbbbb
XRLT_STATUS_DONE
